src/dev/tslint/pick_files_to_lint.ts

```diff
@@ -18,9 +18,17 @@
  */
 
 import { ToolingLog } from '@kbn/dev-utils';
+import minimatch from 'minimatch';
 
 import { File } from '../file';
 
+export const IGNORED_PATHS = ['common/core_api_review/kibana.api.ts'];
+
 export function pickFilesToLint(log: ToolingLog, files: File[]) {
-  return files.filter(file => file.isTypescript() && !file.isFixture());
+  return files.filter(
+    file =>
+      file.isTypescript() &&
+      !file.isFixture() &&
+      IGNORED_PATHS.some(glob => minimatch(file.getRelativePath(), glob, { dot: true }))
```

Comment:
shouldn't we use inversion here? how I can test this script?

Reply:
Good catch! I've pushed a commit to fix it. This script runs as a precommit hook, `files` is an array of staged files, so to test you need to stage some changes with `git add` and then try to commit it.